parity/reports/algod2indexer_all_summary.txt

```diff
@@ -0,0 +1,115 @@
+____________________      61 New Attributes       ____________________
```

Comment:
This report shows all the "essential" difference between **algod** and **indexer** in a purportedly human-friendly format. There are variants of this report that limit the scope to:
* changes of common attributes
* attributes which are not present in **algod** but exist in **indexer**
* attributes which are present in **algod** but missing from **indexer**

However, this report shos _ALL_ the differences (the union of the above)